packages/nvim/src/utils.ts

```diff
@@ -41,7 +41,7 @@ export const shellEnv = (proc = process): NodeJS.ProcessEnv => {
           }, {});
       } catch (e) {
         // Most likely nvim is here.
-        env.PATH = `/usr/local/bin:${env.PATH}`;
+        env.PATH = `/usr/local/bin:/opt/homebrew/bin:${env.PATH}`;
```

Comment:
Hi @bram-dingelstad. Sorry for the late answer and thank you for debugging it.

To test changes locally you can run `yarn build:electron` on your branch. It will build the app and copy it to your Applications folder. Unfortunately, I don't have an M1 Mac to test it.

That `catch` is supposed to be a last resort, we should not get there in the first place. Can you try to run `/bin/zsh -ilc env`, what does it return? Or can you check what is the error it throws on line 43?

Also, could you please reflect your change in the tests here: https://github.com/vv-vim/vv/blob/main/packages/nvim/src/__tests__/utils.test.ts#L41 You can run tests locally with `yarn test` or `yarn test packages/nvim/src/__tests__/utils.test.ts` for a specific file.